pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>io.opentracing.contrib</groupId>
   <artifactId>opentracing-web-servlet-filter-parent</artifactId>
-  <version>0.0.10-SNAPSHOT</version>
+  <version>0.1.0-RC1-SNAPSHOT</version>
```

Comment:
Thought it might be worth bumping the minor version as there are breaking changes.

Reply:
Looks good, perhaps we could merge it to `v0.1.0-RC1` branch. (We have to include the modified `publis.sh`)

Reply:
Is it just a straight copy of the publish.sh from opentracing-java?

Reply:
I will raise a PR against the master, the current version of the script has to be modified.

Reply:
Ah ok - although I've just added it to this PR.

Reply:
https://github.com/opentracing/opentracing-java/pull/198 

This change is missing in the script. Please add it once it's merged. Or we can add it as a separate PR to the master.

Reply:
This can also be left as it is. The version will be changed by release process.